.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
On the one hand, I think this makes a lot of sense, on the other, shouldn't it be something that is part of the main repository, if we're going to do this?

Perhaps leave it out, as not everyone uses editorconfig? (I haven't even heard of it, sounds cool though)